app/src/main/java/com/globant/mariorangel/kotlinonboarding/mvp/view/CountView.kt

```diff
@@ -1,24 +1,65 @@
 package com.globant.mariorangel.kotlinonboarding.mvp.view
 
 import android.app.Activity
-import com.globant.mariorangel.kotlinonboarding.util.bus.OnCountButtonPressedBusObserver
-import com.globant.mariorangel.kotlinonboarding.util.bus.OnResetButtonPressedBusObserver
+import com.globant.mariorangel.kotlinonboarding.mvp.model.CountModel
```

Comment:
The view should not know the model.

Move the constant to another class/interface